src/client/Microsoft.Identity.Client/Http/Retry/IRetryPolicyFactory.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Client.Http.Retry
+{
+    internal interface IRetryPolicyFactory
+    {
+        public enum RequestType
```

Comment:
So this is defining an enum in an interface? This isn't something that we normally do in the code base. Typically, we define enums in their own file and then reference the enum as a property in the interface. Take a look at TokenSource.cs for example